### PR TITLE
[WIP/master] Implement cent/ucent data types.

### PR DESCRIPTION
**Pull request by redstar**
Port of #1355 to the new DDMD frontend.

### CODE REVIEW

File: ddmd/tokens.d [hunk 1]
```diff
@@ -562,8 +562,8 @@ extern (C++) struct Token
     union
     {
         // Integers
-        d_int64 int64value;
-        d_uns64 uns64value;
+        d_int128 int128value;
+        d_uns128 uns128value;
```
Comment (JohanEngelen) [inline ddmd/tokens.d]: Are you worried that math will be noticably slower with this change? (or noticably higher memory usage?)
Once this PR passes the CI systems, ping me so I can test it on Weka's codebase to see if it is noticable in a large project.

Comment (JohanEngelen) [inline ddmd/tokens.d]: "memory usage", lol it's in a union with a struct>=128bits.

**Comment by redstar** [comment]
I'll close this tomorrow. First goal for the hackathon is to implement cent/ucent. :-)

**Comment by dcousens** [comment]
Looking forward to this!

**Comment by redstar** [comment]
@dcousens 
I am working on a PR for DMD. :-)

**Pull request closed without merging**